setup.py

```diff
@@ -69,7 +69,8 @@ def runSetup():
             'six>=1.10.0',
             'future',
             'requests==2.18.4',
-            'docker==2.5.1'],
+            'docker==2.5.1',
+            'python-dateutil'],
```

Comment:
Yes, another dependency, but previously we weren't handling timezones and daylight savings time, so that would eventually cause some nasty bugs.